modin/core/storage_formats/pandas/parsers.py

```diff
@@ -317,6 +317,29 @@ def single_worker_read(cls, fname, *args, reason: str, **kwargs):
             }
         return cls.query_compiler_cls.from_pandas(pandas_frame, cls.frame_cls)
 
+    @staticmethod
+    def get_types_mapper(dtype_backend):
+        """
+        Get types mapper that would be used in read_parquet/read_feather.
+
+        Parameters
+        ----------
+        dtype_backend : {"numpy_nullable", "pyarrow", lib.no_default}
+
+        Returns
+        -------
+        dict
+        """
+        to_pandas_kwargs = {}
+        if dtype_backend == "numpy_nullable":
+            from pandas.io._util import _arrow_dtype_mapping
```

Comment:
Does pandas have something similar to this but public?

Reply:
no